pkg/Library/Node/NodeCreator.js

```diff
@@ -30,7 +28,8 @@ async update({newNodeInfos, graph, nodeTypes}) {
 
 makeNewNode(graph, nodeType) {
   const {id: type, displayName: nodeTypeDisplayName} = nodeType.$meta;
-  const index = this.countNodesOfType(graph.nodes, type) + 1;
+  //const index = this.countNodesOfType(graph.nodes, type) + 1;
```

Comment:
i actually liked it more, when nodes were counted Text1, Text2 etc... do you think the random number are better?

Reply:
Is not better, but the test for prior existence is against the input Graph, and it needs to be against the target Arc instead. This was a stopgap.

Good catch! Thanks for the thorough review.